addons/common/fnc_waitAndExecute.sqf

```diff
@@ -7,12 +7,22 @@ Description:
     Note that unlike PFEH, the delay is in CBA_missionTime not diag_tickTime (will be adjusted for time accl).
 
 Parameters:
-    _function - The function you wish to execute. <CODE>
-    _args     - Parameters passed to the function executing. (optional) <ANY>
-    _delay    - The amount of time in seconds before the code is executed. (optional, default: 0) <NUMBER>
+    _function - The function you wish to execute. [Code]
+    _args     - Parameters passed to the function executing. [Any, defaults to []]
+    _delay    - The amount of time in seconds before the code is executed. [Number, defaults to 0]
+    _getter   - Command evaluated to get the current "time", available options: [String, Code, defaults to "cba_missiontime"]
```

Comment:
`_timeSource`?

Reply:
why even offer string? Just use code that already gives the user the ability to use anything that returns a number, but removes all that crusty code overhead on cba side.